### PR TITLE
Added note on entering right AD domain

### PR DESCRIPTION
**Pull request by ravisha22**
Per FR MFR-M365-18426, added a note that specifically calls out the need to enter the correct domain to avoid deployment failures.

### CODE REVIEW

File: windows-365/enterprise/create-azure-network-connection.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ To create an ANC, you must:
 6. Select **Next**.
 7. For hybrid Azure AD join ANCs, on the **AD domain** page, provide the following information:
     - **AD domain name**: The DNS name of the Active Directory domain that you want to use for connecting and provisioning Cloud PCs. For example, corp.contoso.com.
+ > [!Note]
+ > If your on-premises Acitive Directory environment has more than one domain or parent-child domains, be sure to enter the specific domain in which the Cloud PCs need to be domain joined.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/create-azure-network-connection.md:65]:
```suggestion

  > [!NOTE]
 > If your on-premises Active Directory environment has more than one domain or parent-child domains, be sure to enter the specific domain in which the Cloud PCs need to be domain joined.
 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ravisha22, kindly commit the suggested changes. Many thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**